.github/workflows/release-clients.yml

```diff
@@ -81,7 +81,7 @@ jobs:
 
       - name: Download OpenAPI specification
         if: ${{ !inputs.releaseTag }}
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4.1.7
```

Comment:
> Downloading artifacts that were created from action/upload-artifact@v3 and below are not supported.

https://github.com/actions/download-artifact?tab=readme-ov-file#breaking-changes

we must also upgrade `action/upload-artifact`